graphql_api/types/account/__init__.py

```diff
@@ -1,10 +1,5 @@
 from shared.license import get_current_license
 
-from graphql_api.helpers.ariadne import ariadne_load_local_graphql
+from .account import account, account_bindable
 
-from .account import account_bindable
-
-account = ariadne_load_local_graphql(__file__, "account.graphql")
-
-
-__all__ = ["get_current_license", "account_bindable"]
+__all__ = ["get_current_license", "account_bindable", "account"]
```

Comment:
I noticed this in a lot of GQL files but idk if we actually need get_current_license here, do we?

Reply:
You're right lol. I removed the unnecessary instances and tests are all green.